gensim/models/doc2vec.py

```diff
@@ -580,7 +580,7 @@ def __init__(self, documents=None, dm_mean=None,
         need about 1GB of RAM. Set to `None` for no limit (default).
 
         `sample` = threshold for configuring which higher-frequency words are randomly downsampled;
-                default is 1e-3, useful value is 1e-5.
+                default is 1e-3, values to 1e-5 (or lower) may also be useful.
```

Comment:
`values to` => `values of`. Also, mention the corner case option of "no downsampling" value.